gapic/src/main/com/google/gapid/models/Perfetto.java

```diff
@@ -184,13 +188,14 @@ protected void fireLoadedEvent() {
     public final GpuInfo gpu;
     public final FrameInfo frame;
     public final ImmutableMap<Long, CounterInfo> counters;
+    public final ImmutableListMultimap<Long, Long> gpuCounterGroups;
```

Comment:
nit: Not a big fan of this in such a "global" place, I'd prefer some structure that combines this with the counter info. Mainly because it's not typed, just a map of long-to-longs..., but eh, am ok.